cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -230,7 +230,6 @@ if [ "${CURRENT_STORAGE}" = "etcd3" -a "${TARGET_STORAGE}" = "etcd2" ]; then
   rm -rf "${ROLLBACK_BACKUP_DIR}"
   mkdir -p "${ROLLBACK_BACKUP_DIR}"
   cp -r "${DATA_DIRECTORY}" "${ROLLBACK_BACKUP_DIR}"
-  rm -rf "${DATA_DIRECTORY}"/member/snap/*.snap
```

Comment:
@xiang90 - can you clarify this change? Few weeks ago you or Hongchao told that backup tool requires no ".snap" files. Did it change in the meantime?